DomainContent/AvatarStore/attachmentItemScript.js

```diff
@@ -0,0 +1,156 @@
+                firstGrab = false;
+            }
+        },
+            
+        releaseGrab: function(entityID, args) {
+            var hand = args[0] === "left" ? 0 : 1;
+            var properties = Entities.getEntityProperties(entityID);
+
+            if (Entities.getNestableType(properties.parentID) === "entity") {
+                Messages.sendMessage(messageChannel, "Removed Item :" + entityID);
+                Messages.unsubscribe(messageChannel); 
+                Entities.editEntity(entityID, {parentID: EMPTY_PARENT_ID});
+            }
+
+            var userData = properties.userData;
+            var position = properties.position; 
+            var attachmentData = JSON.parse(userData).Attachment;
+            isAttached = attachmentData.attached;
+
+            if (!isAttached) {
+                _supportedJoints.forEach(function(joint) {
+                    var jointPosition = MyAvatar.getJointPosition(joint);
+                    if (Vec3.distance(position, jointPosition) <= ATTACH_DISTANCE) {
+                        var newEntityProperties = Entities.getEntityProperties(_entityID);
+                        touchJSONUserData(newEntityProperties, function(userData) {
+                            userData.Attachment.attached = true;
+                        });
+                        Entities.editEntity(_entityID, {
+                            parentID: MyAvatar.sessionUUID,
+                            parentJointIndex: MyAvatar.getJointIndex(joint),
+                            userData: newEntityProperties.userData,
+                            lifetime: -1
+                        });
+                        if (ATTACH_SOUND.downloaded) {
+                            Audio.playSound(ATTACH_SOUND, {
+                                position: MyAvatar.position,
+                                volume: AUDIO_VOLUME_LEVEL,
+                                localOnly: true
+                            });
+                        }
+                        Controller.triggerHapticPulse(TRIGGER_INTENSITY, TRIGGER_TIME, hand);
+                    }
+                }); 
+            } else if ( isAttached) {
+                var jointPosition = (properties.parentID === MyAvatar.sessionUUID) ? 
+                    MyAvatar.getJointPosition(properties.parentJointIndex) : 
+                    AvatarList.getAvatar(properties.parentID).getJointPosition(properties.parentJointIndex);
+                if ( Vec3.distance(position, jointPosition) > DETACH_DISTANCE) {
+                    var newDetachEntityProperties = Entities.getEntityProperties(entityID);
+                    touchJSONUserData(newDetachEntityProperties, function(userData) {
+                        userData.Attachment.attached = false;
+                    });
+                    Entities.editEntity(_entityID, {
+                        parentID: EMPTY_PARENT_ID,
+                        lifetime: Entities.getEntityProperties(_entityID, ["age"]).age + 60,
```

Comment:
you, don't need to use the array in get properties when fetching only one property, also please make a constant out of the 60